index.html

```diff
@@ -40,15 +40,16 @@
             </div>
             <div id="nav">
                 <a href="#feature">Features</a>
-                <a href="https://github.com/devarshishimpi/Stone-CSS-Website" target="_blank">Github</a>
+                <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
                 <a href="https://blog.stonecss.com" target="_blank">Blog</a>
                 <a href="./contact.html" >Contact</a>
             </div>
         </div>
         <div id="mobilenav">
             <a href="#feature">Features</a>
-            <a href="https://github.com/devarshishimpi/Stone-CSS-Website" target="_blank">Github</a>
-            <a href="https://blog.stonecss.com" target="_blank">Blog</a>
+
+            <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
+
```

Comment:
```suggestion
 <a href="https://blog.stonecss.com" target="_blank">Blog</a>
```

Please add these changes back.